frontend/assets/icons/viewer/zoom_in.svg.tsx

```diff
@@ -0,0 +1,41 @@
+/**
+ *  Copyright (C) 2024 3D Repo Ltd
+ *
```

Comment:
(this only happens with images)
When I upload an image, then I zoom in twice using the button, and then I try to drag the image, the cursor goes "forbidden" and the image doesn't move until I lift the left-click


https://github.com/3drepo/3drepo.io/assets/43547703/d11e783e-eb33-4c9c-8549-e6347bbda902

Reply:
cant reproduce it  :/